driver/normalizer/annotation.go

```diff
@@ -1,277 +1,448 @@
 package normalizer
 
 import (
-	"errors"
+	"go/token"
 
-	"github.com/bblfsh/go-driver/driver/goast"
 	"gopkg.in/bblfsh/sdk.v1/uast"
-	. "gopkg.in/bblfsh/sdk.v1/uast/ann"
-	"gopkg.in/bblfsh/sdk.v1/uast/transformer"
-	"gopkg.in/bblfsh/sdk.v1/uast/transformer/annotatter"
+	"gopkg.in/bblfsh/sdk.v1/uast/role"
+	. "gopkg.in/bblfsh/sdk.v1/uast/transformer"
 	"gopkg.in/bblfsh/sdk.v1/uast/transformer/positioner"
 )
 
-// Transformers is the of list `transformer.Transfomer` to apply to a UAST, to
-// learn more about the Transformers and the available ones take a look to:
+// Native is the of list `transformer.Transformer` to apply to a native AST.
+// To learn more about the Transformers and the available ones take a look to:
 // https://godoc.org/gopkg.in/bblfsh/sdk.v1/uast/transformer
-var Transformers = []transformer.Tranformer{
-	annotatter.NewAnnotatter(AnnotationRulesExpr),
-	annotatter.NewAnnotatter(AnnotationRulesStmt),
-	annotatter.NewAnnotatter(AnnotationRules),
+var Native = Transformers([][]Transformer{
+	// The main block of transformation rules.
+	{Mappings(Annotations...)},
+	{
+		Mappings(
+			AnnotateIfNoRoles("FieldList", role.Incomplete),
```

Comment:
The `Incomplete` role must not be applied automatically. It's used by the driver programmer to mark roles that doesn't express the full semantic information for any reason (usually because of lack of more specific roles). 

PTAL: https://github.com/bblfsh/documentation/blob/master/proposals/bip-002.md